### ISSUE TITLE
4.2 (11.a) what if subtype has predicates

### ISSUE THREAD

**Issue by nholsti** [pat-rogers/Ada-202x-WG9-Informal-Review#170]
Is this Ramification true if the component subtype of the string has predicates? Example: the subtype contains all and only the vowel letters.

**Comment by ARG-Editor** [comment]
Good catch. I prefixed this with "If no predicates apply to the component
subtype, ...". This was added to the AARM AI (AI12-0005-1).
